src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createTextNode = (type, parentNode, classes, text) => {
+  const node = document.createElement(type);
+
+  node.classList.add(...classes);
+  node.textContent = text;
+  parentNode.append(node);
+
+  return node;
+};
+
+const promise1 = new Promise((resolve) => resolve('Promise was resolved!'));
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+});
+
+logo.addEventListener('click', () => {
+  promise1.then((res) => {
+    createTextNode('div', body, ['message'], res);
+  });
+});
```

Comment:
The `click` event listener on the `logo` element is only resolving `promise1`. If `promise1` fails for any reason, there is no error handling in place. It's a good practice to always handle promise rejections to avoid uncaught errors.